chatops_deployment/ansible/configure.yml

```diff
@@ -15,6 +15,14 @@
       tags:
         - chatops
 
+- name: Set up systemd exporters
+  hosts: all
+  remote_user: ubuntu
```

Comment:
I think you've used `remote_user` throughout, might be worth nipping this whilst we're here and putting

```
cat group_vars/all/vars.yml

ansible_user: ubuntu
```

To just set it for good, then override as required